python27-sys/Cargo.toml

```diff
@@ -12,6 +12,7 @@ homepage = "https://github.com/dgrunwald/rust-python27-sys"
 repository = "https://github.com/dgrunwald/rust-python27-sys.git"
 license = "Python-2.0"
 authors = ["Daniel Grunwald <daniel@danielgrunwald.de>"]
+links = "python27"
```

Comment:
Not 100% sure if I understand the question but here's the explanation .. 

I think this is required in order for the cargo: variables that python27-sys exports to be visible to dependents (such as rust-cpython) as environment variables. cargo seems to use this as a prefix when translating the cargo: vars into environment variable names.

http://doc.crates.io/build-script.html#the-links-manifest-key

For example, in rust-cpython/build.rs above you can see I am unpacking DEP_PYTHON27_PYTHON_FLAGS - well, I never put 'dep_python27' in the cargo: exports at the bottom of python27-sys/build.rs - cargo did. So I think we need to put something here. 

(personally, I think that passing info to dependents should be an orthogonal concept to binding to native libraries - but that doesn't seem to be how it is).